docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -29,6 +29,12 @@ find links to both those resources.
 
 .. _first_steps_1_line_chart:
 
+.. note::
+
+    Bokeh is a Python library that does not have a graphical user interface (GUI). 
```

Comment:
```suggestion
    Bokeh is a Python library that does not have a graphical user interface (GUI).
```

Reply:
Actually can we remove this sentence entirely? I really just think this reads more confusing, and that all we need to state is that the code is to be run as a script with `python`

Reply:
Sounds good